opds.py

```diff
@@ -587,20 +589,21 @@ def groups(cls, _db, title, url, worklist, annotator,
 
         def refresh():
             return cls._generate_groups(
-                _db, title, url, worklist, annotator, facets,
-                search_engine, search_debug
+                _db=_db, title=title, url=url, worklist=worklist,
+                annotator=annotator, pagination=pagination, facets=facets,
+                search_engine=search_engine, search_debug=search_debug
             )
 
         return CachedFeed.fetch(
-            _db, worklist=worklist, facets=facets, pagination=None,
+            _db=_db, worklist=worklist, facets=facets, pagination=pagination,
```

Comment:
Does it make sense to change the order in which parameters are passed and pass `pagination` before `facets`?

Reply:
Good idea.